XRTK.Oculus/Packages/com.xrtk.oculus/Editor/ValidateOculusControllerMappings.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEditor;
+using UnityEngine;
+using XRTK.Oculus.Controllers;
+using XRTK.Utilities.Editor;
+
+namespace XRTK.Oculus
+{
+    [InitializeOnLoad]
+    public class ValidateOculusControllerMappings
+    {
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        static ValidateOculusControllerMappings()
+        {
+            if (!Application.isBatchMode)
+            {
+                EditorApplication.delayCall += CheckControllerMappings;
+            }
+        }
+
+        public static void CheckControllerMappings()
+        {
+            bool result = false;
+            // Test if there is an Native Oculus controller profile registered
+            if (ValidateConfiguration.ValidateControllerProfiles(new[] { typeof(OculusTouchController) }))
```

Comment:
Do we need to create a new array each time?

We could probably make this a local variable.